client/ayon_blender/api/ops.py

```diff
@@ -186,9 +187,12 @@ def _process_app_events() -> Optional[float]:
 
         # Refresh Manager
         if GlobalClass.app:
-            manager = GlobalClass.app.get_window("WM_OT_avalon_manager")
-            if manager:
-                manager.refresh()
+           if not hasattr(GlobalClass.app, "get_window"):
+                GlobalClass.app.processEvents()
+                return TIMER_INTERVAL
+           manager = GlobalClass.app.get_window("WM_OT_avalon_manager")
+           if manager:
+               manager.refresh()
```

Comment:
Why is this changing in this PR? Was there a bug?

Reply:
yes, the pop-up dialog showed correctly when I opened last workfile during launch but it doesn't show up on top correctly when I use workfile tool to open the scene. Maybe we can fix it with different PRs.